File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] a=new int [2];
        int k=0;
		for(int i=0;i<nums.length;i++)
		{
			for(int j=i+1;j<nums.length;j++) 
			{
				if((nums[i]+nums[j])==target)
				{
					a[k]=i;
					a[k+1]=j;
				}
			}
		}
        return a;
		
        
    }
}

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] r=new int[2];
        int index=0;
        for(int i=0;i<nums.Length;i++)
        {
            for(int j=i+1;j<nums.Length;j++){
               if((nums[i]+nums[j])==target)
                {
                    r[index]=i;
                    r[index+1]=j;
                    break;
                }
            }
        }
        return r;
    }
        
}